nipype/interfaces/spm/preprocess.py

```diff
@@ -258,11 +258,13 @@ def _list_outputs(self):
 class ApplyVDMInputSpec(SPMCommandInputSpec):
 
     in_files = InputMultiObject(
-        ImageFileSPM(exists=True),
-        field="data.scans",
+        traits.Either(
+            ImageFileSPM(exists=True), traits.List(ImageFileSPM(exists=True))
+        ),
```

Comment:
Note that `InputMultiObject` is already a single file or a list of files. Is the goal to make it either a single file, a list of files, or a list of lists of files?

Reply:
ok, so regarding your question:

InputMultiPath = InputMultiObject
OutputMultiPath = OutputMultiObject

therefore the syntax I use is the same as other interfaces:

SliceTimingInputSpec
RealignInputSpec
RealignUnwarp
...

also, without this modification my nipype - workflow was failing. 

but you seem to suggest my modification was not needed, and the documentation of the MultiObjects suggests the same. 

but then why my wf was failing? could it be related to the incompatible traits release?
I mean:
- with the newest release, no need for my modification (it would actually be a wrong edit - but also for realign and slicetime interfaces?)
- with the old ones (for example version nipype = "1.8.0" or nipype = "1.8.2") it somehow solves a problem

Reply:
Is the change around L340 enough?

Reply:
@effigies yes! I have done some tests now. so I push anew commit with only changes around line 340?

I guess so.

Reply:
Yes, that would work.

Reply:
can this pull request be accepted then? if not, what remains to be done?